examples/3.x_api/pytorch/nlp/huggingface_models/language-modeling/quantization/weight_only/README.md

```diff
@@ -55,22 +55,23 @@ python run_clm_no_trainer.py \
 ```
 ### Evaluation
 
+> Note: The SRAM_SLICER_SHARED_MME_INPUT_EXPANSION_ENABLED=false is an experimental flag which yields better performance for uint4, and it will be removed in a future release.
```

Comment:
since this is HPU specific, maybe we should consider split it for HPU and others